lib/msal-common/src/response/ServerAuthorizationTokenResponse.ts

```diff
@@ -31,9 +31,9 @@ export type ServerAuthorizationTokenResponse = {
     // Success
     token_type?: AuthenticationScheme;
     scope?: string;
-    expires_in?: number;
-    refresh_in?: number;
+    expires_in?: number | string; // Managed Identity can send an ISO 8601 string instead of a number
```

Comment:
Do we really need to make changes in this base layer of MSAL? Can't you treat this in the MSI layer and adapt the ISO date to expires_in ? and leave this layer unaffected?